velox/expression/ConstantExpr.h

```diff
@@ -57,6 +57,10 @@ class ConstantExpr : public SpecialForm {
     return sharedSubexprValues_;
   }
 
+  bool isNull() const {
```

Comment:
Would this work for complex types, i.e. when expression is created using ConstantExpr(VectorPtr value) constructor?

Reply:
Thank you for catching this! I set a new flag isNull_ in both constructors in the new version.